tests/src/e2e-pull-image.spec.ts

```diff
@@ -0,0 +1,105 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { ElectronApplication, Page } from 'playwright';
+import { _electron as electron } from 'playwright';
+import { afterAll, beforeAll, test, describe } from 'vitest';
+import { expect as playExpect } from '@playwright/test';
+import { existsSync } from 'node:fs';
+import { rm } from 'node:fs/promises';
+
+let electronApp: ElectronApplication;
+let page: Page;
+
+beforeAll(async () => {
+  // remove all videos/screenshots
+  if (existsSync('tests/output')) {
+    console.log('Cleaning up output folder...');
+    await rm('tests/output', { recursive: true, force: true });
+  }
+
+  electronApp = await electron.launch({
+    args: ['.'],
+    recordVideo: {
+      dir: 'tests/output/videos',
```

Comment:
need to be probably a subfolder or how to not remove stuff from the other e2e tests